browser/resources/settings/sources.gni

```diff
@@ -30,6 +30,7 @@ brave_settings_web_component_files = [
   "brave_new_tab_page/brave_new_tab_page.ts",
   "brave_privacy_page/brave_personalization_options.ts",
   "brave_search_engines_page/brave_search_engines_page.ts",
+  "brave_search_engines_page/private_search_engine_list_dialog.ts",
```

Comment:
when you convert this to a `Lit` element, add it (and its `.html.ts` file) to the `non_web_component_files` array